core/blockly.js

```diff
@@ -294,27 +294,20 @@ Blockly.onContextMenu_ = function(e) {
 
 /**
  * Close tooltips, context menus, dropdown selections, etc.
- * @param {boolean=} opt_allowToolbox If true, don't close the toolbox.
+ * @param {boolean=} opt_onlyClosePopups Whether only popups should be closed.
```

Comment:
I would also add something to say specifically that flyouts won't be closed. After learning that people do actually read the jsdoc I'm trying to be more explicit in providing directions/info for developers :)

Reply:
Updated.